applications/GeoMechanicsApplication/tests/cpp_tests/test_mass_matrix.cpp

```diff
@@ -0,0 +1,122 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "custom_elements/plane_strain_stress_state.h"
+#include "custom_utilities/equation_of_motion_utilities.h"
+#include "testing/testing.h"
+#include "tests/cpp_tests/test_utilities/model_setup_utilities.h"
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(CalculateMassMatrix2D6NDiffOrderGivesCorrectResults, KratosGeoMechanicsFastSuite)
```

Comment:
I really like that we have these unit tests to make sure these functions keep doing what they're doing now!